core/client/src/main/java/alluxio/client/file/options/InStreamOptions.java

```diff
@@ -88,6 +91,23 @@ public InStreamOptions setReadType(ReadType readType) {
     return this;
   }
 
+  /**
+   * @return true if incomplete block caching is enabled
+   */
+  public boolean shouldCacheIncompleteBlock() {
+    return mCacheIncompleteBlock;
+  }
+
+  /**
+   * Enable incomplete block caching.
+   *
+   * @return the updated options object
+   */
+  public InStreamOptions enableCacheIncompleteBlock() {
```

Comment:
Same comment I left. it seems the code is not really changed. (did you manage to push the local change?)